core/src/main/java/com/netease/arctic/utils/map/SimpleSpillableMap.java

```diff
@@ -32,32 +31,44 @@
   private static final int RECORDS_TO_SKIP_FOR_ESTIMATING = 200;
   private final long maxInMemorySizeInBytes;
   private final String backendBaseDir;
+  // Size Estimator for key type
+  private final SizeEstimator<K> keySizeEstimator;
+  // Size Estimator for key types
+  private final SizeEstimator<T> valueSizeEstimator;
   private Map<K, T> memoryMap;
   private Optional<SimpleSpilledMap<K, T>> diskBasedMap = Optional.empty();
   private Long currentInMemoryMapSize;
```

Comment:
`long currentInMemoryMapSize` may be better, since the `null` value is not used.

Reply:
Yes